src/Connectors/src/ConnectorBase/DocumentDB/MongoDb/MongoDbConnectorOptions.cs

```diff
@@ -47,7 +47,7 @@ public MongoDbConnectorOptions(IConfiguration config)
 
         public string Database { get; set; }
 
-        public Dictionary<string, string> Options { get; set; }
+        public Dictionary<string, string> Options { get; set; } = new Dictionary<string, string>(StringComparer.InvariantCultureIgnoreCase);
```

Comment:
Does the invariant culture aspect provide value here?

Reply:
Ignoring case is the important part, since we need to specify something I usually just go all the way to invariantcultureignorecase because we want to be pretty flexible about the matching